packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.html

```diff
@@ -8,58 +8,61 @@
 </div>
 
 <div data-ng-if="!$ctrl.loading">
-    <oui-message
-        data-ng-if="!$ctrl.isOrderable"
-        data-type="error">
-        <p class="m-0" data-translate="hosting_dashboard_cdn_order_not_available"></p>
-    </oui-message>
+    <p data-translate="hosting_cdn_order_description"></p>
 
-    <form novalidate
-        name="cdnOrder"
-        data-ng-if="$ctrl.isOrderable"
-        data-ng-submit="$ctrl.makeOrder()">
-        <p data-translate="hosting_cdn_order_text"></p>
+    <oui-stepper
+        data-current-index="$ctrl.currentIndex"
+        data-on-finish="$ctrl.checkout()">
+        <oui-step-form
+            data-header="{{:: 'hosting_cdn_order_step_header_duration' | translate }}"
+            data-editable="$ctrl.isEditable && !$ctrl.checkoutLoading"
+            data-on-focus="$ctrl.resetCart()">
 
-        <oui-field
-            data-label="{{:: 'hosting_cdn_order_duration' | translate }}"
-            data-size="xl">
-            <oui-radio
-                name="duration"
-                data-ng-repeat="duration in $ctrl.durations track by $index"
-                data-model="$ctrl.model.duration"
-                data-value="duration"
-                data-required>
-                <span data-ng-bind="duration | wucDuration:'longDate'"></span>&nbsp;:&nbsp;
-                <span data-ng-bind-html="$ctrl.details[duration].prices | price:$ctrl.user.ovhSubsidiary"></span>
-                <oui-spinner data-size="s" data-ng-if="!$ctrl.details[duration]"></oui-spinner>
-            </oui-radio>
-        </oui-field>
+            <!-- Durations -->
+            <oui-field>
+                <oui-radio
+                    name="interval"
+                    data-ng-repeat="pricing in ::$ctrl.catalogAddon.pricings track by $index"
+                    data-model="$ctrl.interval"
+                    data-value="pricing.interval"
+                    data-required>
+                    <span data-ng-bind=":: '' + pricing.interval | wucDuration:'longDate'"></span>&nbsp;:&nbsp;
```

Comment:
Why using concatenation? `'' + pricing.internal` I mean.

And using `&nbsp;:` is fine in French but it's not true in English for instance.
Maybe we can move it into a translation file, Isn't it?

Reply:
`wucDuration` take a string as input, it's a quick way to convert this number value to a string.

For `:`, I don't think having a translation `{{duration}} :` is a good idea. 
The catalog price that follows is in a component. I don't see any other option than that. Have you a recommandation ?

Reply:
This is specific to locale, so I think a translation is the best thing to do

Reply:
Thanks for the clarification regarding `wucDuration` filter.

Related to the translation, because it is a  specification based on the locale, we do not have other option that providing it through a new translation.

Reply:
Ok, I'll check to put that in a translation :)